src/buildHTML.js

```diff
@@ -66,6 +66,98 @@ export const spliceSpaces = function(children, i) {
     }
 };
 
+const thinspace = {
+    number: 3,
+    unit: "mu",
+};
+const mediumspace = {
+    number: 4,
+    unit: "mu",
+};
+const thickspace = {
+    number: 5,
+    unit: "mu",
+};
+
+export const spacings = {
+    mord: {
+        mop: thinspace,
+        mbin: mediumspace,
+        mrel: thickspace,
+        minner: thinspace,
+    },
+    mop: {
+        mord: thinspace,
+        mop: thinspace,
+        mrel: thickspace,
+        minner: thinspace,
+    },
+    mbin: {
+        mord: mediumspace,
+        mop: mediumspace,
+        mopen: mediumspace,
+        minner: mediumspace,
+    },
+    mrel: {
+        mord: thickspace,
+        mop: thickspace,
+        mopen: thickspace,
+        minner: thickspace,
+    },
+    mopen: {},
+    mclose: {
+        mop: thinspace,
+        mbin: mediumspace,
+        mrel: thickspace,
+        minner: thinspace,
+    },
+    mpunct: {
+        mord: thinspace,
+        mop: thinspace,
+        mrel: thickspace,
+        mopen: thinspace,
+        mclose: thinspace,
+        mpunct: thinspace,
+        minner: thinspace,
+    },
+    minner: {
+        mord: thinspace,
+        mop: thinspace,
+        mbin: mediumspace,
+        mrel: thickspace,
+        mopen: thinspace,
+        mpunct: thinspace,
+        minner: thinspace,
+    },
+};
+
+const tightSpacings = {
```

Comment:
What's the reasoning for distinguishing these from `spacings`?

Reply:
The spacings are different when we're in `\scriptstyle` or `\scriptscriptstyle`.  I'm open to suggestions of other ways to organize these alternate values.